Packs/CiscoUmbrellaReporting/pack_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Cisco Umbrella Reporting",
+    "description": "Use Cisco Umbrella's Reporting to monitor your Umbrella integration and gain a better understanding of your Umbrella usage. Gain insights into request activity and blocked activity, determining which of your identities are generating blocked requests. Reports help build actionable intelligence in addressing security threats including changes in usage trends over time. The Umbrella Reporting v2 API provides visibility into your core network and security activities and Umbrella logs. This integration was integrated and tested with version 2 of Cisco-umbrella-reporting.",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
```

Comment:
In the design doc the category is: Network Security. 
@meital-aizen - what is the right category here?

Reply:
@ShacharKidor Correct, it should be changed to Network Security. FYI @RamboV